### DISCUSSION TITLE
建议增加创作者激励和多种收费模式

### DISCUSSION THREAD

**Discussion by Luyi365** [lobehub/lobehub#4403] [Ideas]
### 创作者激励
希望能够对于该项目有贡献的创作者们进行积分或会员时长的奖励，例如：根据创建的助手热度来奖励、对插件的开发者进行奖励等等。
奖励意味着激励，会为项目和创作者们带来双赢。

### 多种收费模式
我在使用lobe-chat时考虑将其他厂商的API接入进来，但我发现我不仅要为API付费，同时也要为lobe-chat购买会员。实际上当我拥有一些API后，lobe-chat对我来说更像是一个chat的聚合工具，作为API数据传递的媒介，在这种情况下我不会消耗lobe-chat服务器太多流量。
因此，当我同时购买API和lobe-chat是十分昂贵的，性价比变得很低。所以我希望能够推出对某项服务进行收费的模式，例如：模型服务、云服务等等。

_所有建议，都是希望lobe-chat变得更好~_

**Comment by arvinxx** [comment]
提议很好，这个月我们会增加一种新的付费订阅计划，只包含基础的平台服务费，但此时可以使用自己的 api。

创作者激励这个也有对应的想法，但暂时没这么快，敬请期待~

**Comment by arvinxx** [comment]
@Luyi-2333 hello，自助版订阅方案已经在 Cloud 版上线

**Reply by Luyi365** [reply]
OK，已经订阅了
